src/renderer/src/global.css

```diff
@@ -70,6 +70,7 @@
 * {
   margin: 0;
   padding: 0;
+  user-select: none;
```

Comment:
Should this change be here?

Reply:
I feel like it's a sane default for an electron app, and idk about making another PR just for this one line change. should have mentioned this though

Reply:
Like I mentioned before I'd rather not set this without either having a context menu option to copy certain text elements or by changing those elements (e.g. song title and artist) to be selectable.